### PR TITLE
Added Internationalization Service(ngx-translate)

### PR DESCRIPTION
**Pull request by IvanShterev**

### CODE REVIEW

File: ionic-angular-template/src/app/home/home.page.html [hunk 1]
```diff
@@ -15,6 +15,6 @@
 
   <div id="container">
     <strong>Ready to create an app?</strong>
-    <p>Start with Ionic <a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/components">UI Components</a></p>
+    <p>{{ 'welcometitle' | translate }}<a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/components">UI Components</a></p>
```
Comment (ochikov) [inline ionic-angular-template/src/app/home/home.page.html]: Let's use `kebab-case` for the labels. For example: `welcome-title`

File: ionic-angular-template/src/assets/i18n/en.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "welcometitle": "Welcome :) "
```
Comment (ochikov) [inline ionic-angular-template/src/assets/i18n/en.json]: Let's use `kebab-case` for the labels. For example: `welcome-title`

File: ionic-angular-template/src/app/home/home.page.ts [hunk 3]
```diff
@@ -7,6 +8,8 @@ import { Component } from '@angular/core';
 })
 export class HomePage {
 
-  constructor() {}
+  constructor(private translate: TranslateService) {
+    translate.setDefaultLang('en');
```
Comment (ochikov) [inline ionic-angular-template/src/app/home/home.page.ts]: This should be in the app.ts file

File: ionic-angular-template/src/app/home/home.module.ts [hunk 4]
```diff
@@ -16,7 +24,14 @@ import { HomePage } from './home.page';
         path: '',
         component: HomePage
       }
-    ])
+    ]),
+    TranslateModule.forRoot({
```
Comment (ochikov) [inline ionic-angular-template/src/app/home/home.module.ts]: Can we move this to the app.module.ts

Comment (IvanShterev) [reply ionic-angular-template/src/app/home/home.module.ts]: I got this error "The pipe 'translate' could not be found". I checked it and found out that my home page is lazily loaded and therefore I have to add these configs to home.module as well.

Comment (IvanShterev) [reply ionic-angular-template/src/app/home/home.module.ts]: Can I do a shared module instead of adding these configs every time I need it ?

Comment (ochikov) [reply ionic-angular-template/src/app/home/home.module.ts]: I think that the initialization should be in the app.module.ts and then you should import in every module.ts the `TranslateModule`

For example: `import { TranslateModule } from '@ngx-translate/core';`

Then: 
`@NgModule({
  imports: [
    CommonModule,
    FormsModule,
    ReactiveFormsModule,
    TranslateModule,
    IonicModule,
    SharedModule,
    RouterModule.forChild(routes)
  ],
  declarations: [ForgotPasswordPage, EnterEmailComponent]
})`

**Pull request merged**